boards/microbit-v2/include/periph_conf.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_microbit_v2
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the BBC micro:bit v2
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "board.h"
+#include "periph_cpu.h"
+#include "cfg_clock_32_1.h"
+#include "cfg_timer_default.h"
+#include "cfg_rtt_default.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Enable the internal DC/DC converter
+ */
+#define NRF5X_ENABLE_DCDC
+
```

Comment:
Does the RTT work with

    #define CLOCK_LFCLK 0

to use the internal RC oscillator

Reply:
Thanks, it works when LFCLK is derived from HFCLK.